.gitignore

```diff
@@ -0,0 +1,78 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
No need to polute the .ignore file with all this. Please revert to previous version.